index.html

```diff
@@ -1,12 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" />
 </head>
+
 <body>
-    
+    <section class="city-name-container">
+        <h1>Weather Report</h1>
+        <h2>for the lovely city of...</h2>
+        <h2 id="newCity">Choose a city</h2>
+    </section>
+
+    <section class="search-container">
+        <input id="cityToSearch" type="text">
+        <button id="changeCity">Change City</button>
+        <button id="resetButton">Reset City</button>
+    </section>
+
+    <section class="temp-display">
+        <button id="tempUp">⬆</button>
+
+        <section id="currentTemp">
+            <p id="temp">60℉</p>
+            <h3>Current Temp</h3>
+        </section>
+
+        <button id="tempDown">⬇</button>
+
+        <button id="getRealTimeTemp">Get Actual Current temperature</button>
+    </section>
+
+    <section id="sky_landscapeContainer">
+        <h2>Weather Garden</h2>
```

Comment:
This should be a `h3` like the previous section title because it sits under your `h2` from above.  (And elements at the same level semantic nesting should use the same level of heading tag.)